src/components/AddToClipboard.svelte

```diff
@@ -26,6 +26,7 @@
 
   const copy = () => {
     const app = new Clipboard({
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       target: document.getElementById('clipboard')!,
       props: { copyValue },
```

Comment:
```suggestion
    const clipboardElement = document.getElementById('clipboard');
    if (!clipboardElement) return;
    const app = new Clipboard({
      target: clipboardElement,
      props: { copyValue },
```

Thinking there are some pretty simple ways to avoid having to disable eslint. thoughts on something like the above?

Reply:
Hmm... That does remove it, but it suppresses the error. In theory the button shouldn't exist if the element couldn't be found.

If you want, throw in an issue to clean it up.